src/shared/components/Button/styles.ts

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+import Colors from '../../colors';
+
+export enum ButtonVariant {
+  PRIMARY,
+  PRIMARY_SMALL,
+  OUTLINE,
+  DARK,
+}
+
+type StyleProps = {
+  variant: ButtonVariant;
+};
+
+const bigSizeStyle = `
+  height: 54px;
+  font-size: 16px;
+  line-height: 20px;
+  padding: 0 30px;
+`;
+
+const primaryBaseStyle = `
+  border: 2px solid ${Colors.ORANGE_DARK};
+  background-color: ${Colors.ORANGE_DARK};
+  color: ${Colors.WHITE};
```

Comment:
I think you need to add minimal animation for all type buttons

Reply:
done